frames/src/main/java/com/checkout/frames/screen/billingaddress/billingaddressdetails/models/BillingAddress.kt

```diff
@@ -5,7 +5,7 @@ import com.checkout.tokenization.model.Address
 import com.checkout.tokenization.model.Phone
 
 internal data class BillingAddress(
-    val name: String? = null,
+    var name: String? = null,
```

Comment:
we should avoid using `var` here, same for `phone`, to avoid any side effects. 

Left a comment for `provideBillingAddressDetails` for how could it be improved.